Resources/Prototypes/_EinsteinEngines/NPCs/carp.yml

```
@@ -0,0 +1,94 @@
+      makeSentient: true
+      name: ghost-role-information-space-goldfish-name
+      description: ghost-role-information-space-goldfish-description
+      rules: ghost-role-information-freeagent-rules
+      mindRoles:
+      - MindRoleGhostRoleFreeAgent
+    - type: GhostTakeoverAvailable
+    - type: Sprite
+      drawdepth: Mobs
+      sprite: Mobs/Aliens/Carps/goldfish.rsi
+      layers:
+      - map: [ "enum.DamageStateVisualLayers.Base" ]
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)